packages/mui-material/src/Card/Card.spec.tsx

```diff
@@ -17,17 +17,11 @@ function CardTest() {
           expectType<React.MouseEvent<HTMLDivElement, MouseEvent>, typeof event>(event);
         }}
       />
-      <Card
-        component="a"
-        href="test"
-        onClick={(event) => {
```

Comment:
Why was this assertion removed?

Reply:
@michaldudak after adding `{ component?: React.ElementType }` to components, are types of event handlers expected to be inherited from `component`?

for example in below code - is type of event expected to be `React.MouseEvent<HTMLDivElement, MouseEvent>` or `React.MouseEvent<HTMLAnchorElement, MouseEvent>`. Right now `event` type is `React.MouseEvent<HTMLDivElement, MouseEvent>`

```
<Card
component='a'
        onClick={(event) => {
         
        }}
      />
```

Reply:
Yes - the `*Props` types are not used when you instantiate a component (`OverridableComponent` does not use `*Props`), so there should be no changes in this matter.

Reply:
@sai6855 I'm working on a fix for this.